ArduCopter/config.h

```diff
@@ -140,8 +140,8 @@
 
 //////////////////////////////////////////////////////////////////////////////
 // Precision Landing with companion computer or IRLock sensor
-#ifndef PRECISION_LANDING
- # define PRECISION_LANDING ENABLED
+#ifndef AC_PRECLAND_ENABLED
+ # define AC_PRECLAND_ENABLED ENABLED
```

Comment:
```suggestion
```

This is done i nthe library header file. Just remove all of this block.


Similarly in other vehicles